README.rst

```diff
@@ -13,7 +13,7 @@ shared object model. In order to be as lightweight a dependency as possible,
 Neo is deliberately limited to represention of data, with no functions for data
 analysis or visualization.
 
-Neo is used by a number of other software tools, including 
+Neo is used by a number of other software tools, including
 SpykeViewer_ (data analysis and visualization), Elephant_ (data analysis),
```

Comment:
I think that spykeviewer is not maintain anymore (like OpenElectrophy) we should remove from the list. No ?